tools/python/openlr/quality.py

```diff
@@ -143,32 +149,38 @@ def parse_segments(tree, limit):
         # within limit are considered accurate, so golden path should be equal
         # matched path.
         golden_route = parse_route(s.find('GoldenRoute')) or matched_route
-        if not matched_route and not golden_route:
-            raise ValueError('At least one of golden route or matched route should be present')
+        yield Segment(segment_id, golden_route, matched_route)
```

Comment:
In the this version param order has changed?

Reply:
Yep